orc8r/cloud/go/services/certifier/obsidian/models/swagger.v1.yml

```diff
@@ -0,0 +1,153 @@
+---
+swagger: '2.0'
+
+magma-gen-meta:
+  go-package: magma/orc8r/cloud/go/services/certifier/obsidian/models
+  dependencies:
+    - 'orc8r/cloud/go/models/swagger-common.yml'
+  temp-gen-filename: certifier-swagger.yml
+  output-dir: orc8r/cloud/go/services/certifier/obsidian
+  types:
+    - go-struct-name: User
+      filename: user_swaggergen.go
+
+
+info:
+  title: Certifier's User Management
+  description: Certifier REST APIs
+  version: 1.0.0
+
+basePath: /magma/v1
+
+paths:
+  /user:
+    get:
+      responses:
+        "200":
+          description: Success
+          schema:
+            type: array
+            items:
+              type: string
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+      summary: List all usernames
+      tags:
+        - User
+    post:
+      parameters:
+        - name: user
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/user_with_policy'
+      responses:
+        "201":
+          description: Success
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+      summary: Create a user
+      tags:
+        - User
+  /user/{username}:
+    put:
+      parameters:
```

Comment:
Hmm I think it might be better if this just handles the password. The only other field in the user are the tokens. I feel like it would not make sense for the admin to put malformed tokens that are not associated with any policy here. The admin can manage the tokens from the separate token endpoints. Lmk what you think.

Reply:
I think the value of splitting up /user/{username} and /user/{username}/tokens is that the "user" doesn't contain any of the tokens

I.e. a user is (currently) just a username and password. In which case we wouldn't have to worry about this

Reply:
Then how do users get any available set of tokens that they can use if not stored with the user object?!